crowdfunding_compassion/controllers/projects_controller.py

```diff
@@ -30,6 +30,7 @@ def get_projects_list(self, **kwargs):
            auth="public",
            type="http",
            method='POST',
+           csrf=False,
```

Comment:
Please remove this, it will introduce security issues. It works fine on the devel server.